### ISSUE TITLE
Staging Review finding: Heading structure does not match page hierarchy

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#129388]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ursa Minor
**Product:** Travel Pay Claims
**Feature:** Complex Claims v1

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

The heading structure does not match the page hierarchy. There is a visual hierarchy, which isn't present programmatically, resulting in a flat and confusing content organization. 

```
<h2>Your travel claims</h2>
<h2>Filter and sort travel claims</h2> (this should probably be an H3)
<h2>Showing 1 ‒ 10 of 49 claims, sorted by newest first.</h2> (this probably shouldn't be a heading)
```

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="566" height="504" alt="headings" src="https://github.com/user-attachments/assets/54cc1e74-b9ff-415f-aa31-3421ee636ada" />

</details>

## Recommended action

Revisit the heading structure to ensure that the programmatic hierachy matches the visual hierarchy. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by pmclaren19** [comment]
This is related to the claim list and not complex claims. Moving to backlog.
